### DISCUSSION TITLE
O que vocês acham de certificaçao JavaScript?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#491] [Discussão]
Conheci o teste http://certificacao.imasters.com.br/ durante a intercon em 2016, eu nao passei haha e ouvi alguns amigos fazendo algumas criticas, que achei bastante construtivas.

Gostaria de saber de voces que chegaram a testar o que voces acharam dessa certificaçao especificamente, e o que voces acham de certificaçoes at all.

Abs.

**Comment by felipefialho** [comment]
Deixei o titulo mais amplo o/

**Comment by MarcoWorms** [comment]
Gostaria de dar minha crítica, fiz e passei no teste mas achei ele completamente falho. Ele não avalia nenhuma "good part" do javascript. Não há questões boas sobre closures, nem ES6, grande parte do desafio consiste em pegadinhas com "this". Eu nunca usaria essa certificação como métrica para uma contratação ou ao menos para medir o nível de alguem em JS. Os problemas do mundo real de JS não estão aí.

@LFeh eu acho que o felquis se referiu especificamente a essa certificação pois nós fizemos o teste dela na Intercon e ficamos bastante decepcionados, espero que ninguém leve esse "certificado" em consideração na hora de ponderar candidatos.

**Comment by WandersonAlves** [comment]
Po, não conhecia. É 0800, então talvez seja valido, irei tentar mais tarde e darei uma opinião mais ampla

**Comment by AlexsandroSA** [comment]
@felquis boa noite cara. Conhecia o curso mas nunca cheguei a fazer, também tive essa dúvida.

@WandersonAlves  uma opção 0800 e um pouco mais ampla é o [freecodecamp](https://www.freecodecamp.com/). Inicie essa curso/certificação um tempo atrás e estou gostei muito, sai bem do iniciante(o que pode ser chato no inicio) e vai até o intermediar. 

Tem alguns desfios/projetos [fiz alguns](https://codepen.io/AlexsandroSA/#) tem muito mais veja a [lista de completa de desafios/projetos do curso](https://www.freecodecamp.com/map).  Vale a pena dá uma olhada lá. 

Ele é em Inglês. :)

**Comment by WandersonAlves** [comment]
@AlexsandroSA que show, eu só lia o medium do freecodecamp, nunca parei pra ir no site deles mesmo. Darei uma olhada.

Sobre a prova do iMasters, tirei 60% dos 80% necessários. Muita "casca de banana" na prova, hora de estudar mais!

**Comment by felisio** [comment]
Só para acrescentar existe também a certificação da microsoft em HTML5, Javascript e CSS3

https://www.microsoft.com/pt-br/learning/exam-70-480.aspx

**Reply by atmmoreira** [reply]
Vale a pena irar essa certificação?

**Reply by felisio** [reply]
Certificações sempre valem a pena em uma área como a nossa em que trabalhamos com base em tecnologias criadas por empresas para suprir as necessidades dos seus produtos. Agora se vale a pena tu que tem que decidir com base nas suas necessidades de mercado. No meu caso, na epoca que fiz a empresa que trabalhava exigiu para eu poder tentar uma posição melhor, mais no mercado de trabalho no geral não fez muita diferença hahaha.

**Reply by atmmoreira** [reply]
Opa @felisio pelo que li no site, eles dizem que foi reformado essa certificação. Ae fiquei sem entender. E uma pergunta, onde podemos conseguir certificações para a área de frontend especificamente Javascript.

A mensagem do site é "Este exame foi reformado em 31 de janeiro de 2021. Para obter as opções disponíveis atualmente, consulte a página Navegue pela página de Certificações e Exames."

Existe algum livro para essa documentação atualizado. Tenho interesse em fazer, bem como a de Node.

Achei essas:

: Certificações
Javascript, HTML e CSS
CIW JavaScript Specialist - https://www.ciwcertified.com/ciw-certifications/web-development-series/javascript-specialist
Exame 98-382: Introduction to Programming Using JavaScript - https://docs.microsoft.com/pt-br/learn/certifications/exams/98-382
Exame 98-383: Introduction to Programming Using HTML and CSS - https://docs.microsoft.com/pt-br/learn/certifications/exams/98-383
Exame 70-480: Programming in HTML5 with JavaScript and CSS3 - https://docs.microsoft.com/pt-br/learn/certifications/exams/70-480

Node - Livro referência - Node Cookbook
OpenJS Node.js Application Developer (JSNAD) - https://training.linuxfoundation.org/certification/jsnad/


att

**Reply by felisio** [reply]
Geralmente isso acontece quando vai sair uma nova versão da certificação ou quando a mesma será descontinuada. Acredito que seja a primeira opção. Mais em relação a outras empresas certificadoras especifico para front end de cabeça assim eu não sei, só dando uma pesquisada mesmo. Na epoca que fiz Microsoft era que estava no top em relação a isso

**Reply by atmmoreira** [reply]
Blza. Obrigado.
